{{cookiecutter.repo_name}}/README.txt

```diff
@@ -20,9 +20,28 @@ Getting Started
 
     env/bin/pip install -e ".[testing]"
 
-- Configure the database.
-
-    env/bin/initialize_{{ cookiecutter.repo_name }}_db development.ini
+- Initialize the database.
+
+    - This cookiecutter provides support for initializing the database either
+    - (a) with migrations through alembic, or
+    - (b) without migrations.
+    
+    - (a) Initialize the database with migrations through alembic:
+    - First use alembic to generate revisions.
+
+        env/bin/alembic -c development.ini revision --autogenerate -m "init"
+
+    - upgrade to that revision:
+
+        env/bin/alembic -c development.ini upgrade head
+    
+    - Later you can run database migrations.
+    - Also there is commented code that can be used to
+    - initialize the database directly to the latest revision.
+    - {{cookiecutter.repo_name}}/scripts/initializedb.py
```

Comment:
I do not like leaving commented code in here for an alternate workflow. There should either be 1) an option in the cookiecutter.json to select with or without alembic or 2) we update the cookiecutter to only support the alembic workflow.

I prefer option 2.

Reply:
Without migrations, tutorial users are left to figure out on their own how to update their database after initialization. Users may follow the example in the tutorial and continually update the `initializedb.py` script only until they finally realize it's a bad idea. I agree with @mmerickel to support only the Alembic workflow.

Reply:
So there should be only option 2) alembic way with migrations. Is this final opinion?

Reply:
@ibeex yes, please. That is the consensus.

Reply:
I am hawing trouble with initialize_db.py
after removing (b) option from readme
```
- Initialize the database.

    - This cookiecutter provides support for initializing the database
    - with migrations through alembic
    
    - Initialize the database with migrations through alembic:
    - First use alembic to generate revisions.

        env/bin/alembic -c development.ini revision --autogenerate -m "init"

    - upgrade to that revision:

        env/bin/alembic -c development.ini upgrade head
    
    - Later you can run database migrations.
```
I don't see need for `initialize_db` all db should be handeled with `env/bin/alembic ...` so I would remove `initialize_db.py`. That way suer can folow http://alembic.zzzcomputing.com/en/latest/tutorial.html#running-our-second-migration doc's for later workflow.

Reply:
I think this approach is ok with me for handling schema changes. In my projects I usually have two commands `db:migrate` and `db:load-fixtures`. Performing an `alembic upgrade head` is equivalent to `db:migrate` whereas right now we do not have anything for loading fixtures - a pattern I think some people have been putting into the `initialize_db` script. This is a good opportunity to add that missing pattern but I'm ok waiting until after this is merged to do that if we wish.